demo/src/screens/componentScreens/ColorSwatchScreen.tsx

```diff
@@ -61,10 +65,16 @@ export default class ColorSwatchScreen extends Component {
             Single ColorSwatch
           </Text>
           <View row>
-            <ColorSwatch selected={selected} onPress={this.onPress}/>
             <View>
-              <ColorSwatch selected color={Colors.$backgroundMajorLight}/>
-              <Text>Disabled</Text>
+              <ColorSwatch selected={selected} onPress={this.onPress}/>
+            </View>
+            <View>
+              <ColorSwatch color={Colors.$backgroundMajorLight}/>
+              <Text text90R>Disabled</Text>
+            </View>
+            <View>
+              <ColorSwatch unavailable onPress={this.unavailableOnPress} color={Colors.yellow10}/>
```

Comment:
unavailable Swatch should be pressable?

Reply:
yes, according to the guidelines it can be pressable (passing onPress) or disabled (when the user doesn't pass onPress).

Reply:
Ok, but there is no indication to the user this Swatch was pressed... not even active opacity...

Reply:
Yes I know, I told Ethan and Diana about it.